benchmarks/newton_solver_benchmark_set/spiegelman_et_al_2016/drucker_prager_compositions.cc

```diff
@@ -126,7 +126,7 @@ namespace aspect
 
         /**
          * Defining a minimum strain rate stabilizes the viscosity calculation,
-         * which involves a division by the strain rate. Units: $1/s$.
+         * which involves a division by the strain rate. Units: \\si{\\per\\second}.
```

Comment:
This is run through doxygen, which will only get this right if you put it between `$...$` (it isn't given to latex otherwise).

Reply:
Argh! Thanks! I've searched through for all other instances of this (there were only a few).